src/pages/batch-create-offers/BatchCreateOffers.tsx

```diff
@@ -0,0 +1,273 @@
+          existingHashes.push(hash);
+        }
+      }
+    }
+    const existingOffers: offers.CreateOfferArgs[] = [];
+    const notExistingOffers: offers.CreateOfferArgs[] = [];
+    const existing: Data[] = [];
+    const notExisting: Data[] = [];
+    for (let i = 0; i < offersList.length; i++) {
+      const offer = offersList[i];
+      const metadata = offerMetadatas.isSuccess
+        ? offerMetadatas.data?.[i]
+        : undefined;
+      const variation = metadata?.variations?.map((v) => v.option).join(" ");
+      const name = metadata
+        ? `${offer.metadataHash} (${metadata.name} ${variation || ""})`
+        : offer.metadataHash;
+      if (existingHashes.includes(offer.metadataHash)) {
+        existingOffers.push(offer);
+        existing.push({
+          name,
+          value: (
+            <>
+              <Typography tag="p">offer already exists</Typography>
+            </>
+          )
+        });
+      } else {
+        notExistingOffers.push(offer);
+        notExisting.push({
+          name,
+          value: (
+            <>
+              <Typography tag="p">to be created</Typography>
+            </>
+          )
+        });
+      }
+    }
+    setTableData([...notExisting, ...existing]);
+    setOffersToBeCreated(notExistingOffers);
+  }, [
+    offersList,
+    sellerOffers.isSuccess,
+    sellerOffers.data,
+    offerMetadatas.isSuccess,
+    offerMetadatas.data
+  ]);
+
+  const handleCreateOffers = async () => {
+    // TODO: take into account the max number of offers that can be created in a batch transaction
+    try {
+      await createOffers({
+        sellerToCreate: null,
+        offersToCreate: offersToBeCreated,
+        tokenGatedInfo: null, // TODO: add token gated info
+        conditionDecimals: decimals,
+        onGetExchangeTokenDecimals: setDecimals,
+        onCreatedOffersWithVariants: () => {
+          toast((t) => (
+            <SuccessTransactionToast
+              t={t}
+              action={`${offersToBeCreated.length} offers created`}
+            />
+          ));
+        },
+        onCreatedSingleOffers: ({ offer: createdOffer }) => {
+          toast((t) => (
+            <SuccessTransactionToast
+              t={t}
+              action={`Created offer: ${createdOffer?.metadata?.name}`}
+            />
+          ));
+        }
+      });
+    } catch (error: unknown) {
+      // TODO: FAILURE MODAL
+      console.error("error->", (error as { errors: unknown }).errors ?? error);
+      const hasUserRejectedTx =
+        "code" in (error as { [key: string]: unknown }) &&
+        (error as { code: string })?.code === "ACTION_REJECTED";
+      if (hasUserRejectedTx) {
+        showModal("TRANSACTION_FAILED");
+      } else {
+        showModal("TRANSACTION_FAILED", {
+          errorMessage: "Something went wrong"
+        });
+      }
+    }
+  };
+
+  return (
+    <>
+      <Formik
+        initialValues={{ files: "" }}
```

Comment:
add validationSchema if possible

Reply:
Adding a validation schema here breaks the logic I've implemented.
At the moment the Create Offers button is the Form submission command. But managing its status (being enable or disable) is outside of the Form scope (it requires to read the selected file, to parse all offers there, and check if there are some offers to be created - we disable if all offers are already existing).
That's why I'm managing the validity my own way